### ISSUE TITLE
FPS drops with too many fireball entities

### ISSUE THREAD

**Issue by KevinMagers** [xJon/The-1.7.10-Pack#1688]
So im running a small private server with this pack v0.9.8 and me and anyone who has logged in always seemed to have major FPS drops when fighting a group of blazes or clerics from thaumcraft, testing this out with blazes myself i noticed that the frames are worse the more fireballs there are mid flight. I belive this has to be a client-side issue since its fps based, ive allocated for java from 3.5gb to 8gb while performance slightly increases with more ram, I can still get fps to near zero after triggering a volley from 3 freshly spawned blazes.

**Comment by clienthax** [comment]
No system specs, try turning off particles.

**Comment by KevinMagers** [comment]
As I mentioned this happens to anyone who logs in so specs vary. my personal specs for this computer AMD-FX 9370 cpu, gtx 1070 gpu 12gb ram.
particles have no real bearing on my performance off or on
also has anyone been able to fight the ur-ghast in twilight without cheesing, it is impossible with these frame drops, as the ur-ghast and other ghasts near it shoot projectile fireballs
UPDATE: I've found a clue, I believe this issue has to do with dynamic lights. I tested this out with a fire charge and dispenser to show that an inflight fire charge attempts to update every set of blocks it passes over with illumination. This must mean the issues is the server instead of the client right? since if have dynamic lights disabled within the client, the server is the one updating light levels since it determines mobs spawns... BRB looking into disabling dynamic lights server side
